dev-requirements.txt

```
@@ -1,17 +1,17 @@
-black==24.10.0; python_version > "3.8"
-black==24.8.0; python_version <= "3.8"
+black==25.1.0; python_version > "3.8"
+black==25.1.0; python_version <= "3.8"
 isort; python_version < "3.8"
-isort==5.13.2; python_version >= "3.8"
+isort==6.0.1; python_version >= "3.8"
 pre-commit; python_version <= "3.8"
 pre-commit==4.1.0; python_version > "3.8"
-codespell==1.16.0
+codespell==v2.4.1
 flake8; python_version < "3.8"
-flake8==7.1.1; python_version >= "3.8"
-bandit==1.7.10; python_version <= "3.8"
-bandit==1.8.0; python_version > "3.8"
+flake8==7.1.2; python_version >= "3.8"
+bandit==1.8.3; python_version <= "3.8"
+bandit==1.8.3; python_version > "3.8"
```

```suggestion
bandit; python_version <= "3.8"
bandit==1.8.3; python_version > "3.8"
```

Same deal, keep versionless for 3.8 so it'll get whatever is actually available.  We'll probably want to remove the line entirely eventually but I don't feel like thinking about that today.